lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -56,7 +56,8 @@ def get_rubric_config(self, request, usage_id):
         """
         data = {'target_rubric_block_id': usage_id}
         response = call_xblock_json_handler(request, usage_id, 'get_rubric', data)
-        return json.loads(response.content)
+        response_data = json.loads(response.content)
+        return response_data['rubric']
```

Comment:
Note that I'm delaying more robust error handling to a follow-up story, https://openedx.atlassian.net/browse/AU-469